spark/src/test/scala/org/apache/spark/sql/delta/DeletionVectorsTestUtils.scala

```diff
@@ -38,12 +38,14 @@ trait DeletionVectorsTestUtils extends QueryTest with SharedSparkSession {
   def enableDeletionVectors(
       spark: SparkSession,
       delete: Boolean = false,
-      update: Boolean = false): Unit = {
-    val global = delete | update
+      update: Boolean = false,
+      merge: Boolean = false): Unit = {
+    val global = delete | update | merge
```

Comment:
Nit (pre-existing): Why not use `||` instead of `|`? It's kinda of rare to see `|` used with booleans.